crates/ethereum/node/src/node.rs

```diff
@@ -46,6 +46,7 @@ impl EthereumNode {
         EthereumNetworkBuilder,
         EthereumExecutorBuilder,
         EthereumConsensusBuilder,
+        EthereumEngineValidatorBuilder,
```

Comment:
this is fine for now, I think we'd like to not have this as a component because this is engine API specific, but for now this is totally fine since it unblocks things and the change is not invasive